lib/order.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :current_order
+
+  def initialize(menu)
```

Comment:
def initialize(menu= Menu.new) ... you could pass a new instance of the menu class, if someone just does order = Order.new the @menu will refer to your default menu, but you can always do order =Order.new({new_menu_item: 5}) to add in a new menu

This way you don't have to make a menu first then pass it through, it combines these two steps